.github/workflows/start-release.yml

```
@@ -152,14 +152,10 @@ jobs:
         if: steps.check-existing.outputs.branch-exists == 'false'
         run: |
           git switch --create "${{needs.variables.outputs.branch}}"
-          # Tagging the branching point so that the commits on the main branch
-          # be correctly described.
+          git commit --allow-empty --message \
+            "Initial commit to release branch ${{needs.variables.outputs.release}}"
```

In good [StackRox tradition](https://spaces.redhat.com/display/StackRox/How+to+manually+manage+an+Upstream+StackRox+release#HowtomanuallymanageanUpstreamStackRoxrelease-Prepwork&branch), I suggest to tweak the message to:
```suggestion
          git commit --allow-empty --message \
            "Empty commit to diverge ${{needs.variables.outputs.release}} from master"
```

There's nothing wrong with the current message, I just somehow feel attached to the old one :-)